learning_journal/views/views.py

```diff
@@ -44,3 +53,22 @@ def edit_existing(request):
                             creation_date=entry.creation_date)
         return HTTPFound(location=request.route_url("index"))
     return {"entry": entry}
+
+
+@view_config(route_name="login",
+             renderer="../templates/login.jinja2",
+             permission=NO_PERMISSION_REQUIRED)
```

Comment:
In case you are not aware, remember that you can also *not* declare a default permission, and then these uses of `NO_PERMISSION_REQUIRED` are not needed.  It also works to go this way, so no foul.